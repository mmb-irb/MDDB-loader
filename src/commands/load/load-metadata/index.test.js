const loadMetadata = require(__dirname);

describe('loadMetadata', () => {
  const spinnerRef = { current: null };
  test('basic', async () => {
    expect(
      await loadMetadata('metadata', `${__dirname}/__fixtures`, spinnerRef),
    ).toMatchSnapshot();
  });
});
