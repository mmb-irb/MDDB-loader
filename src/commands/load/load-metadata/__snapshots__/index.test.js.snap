// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`loadMetadata basic 1`] = `
Object {
  "AUTHORS": "Me, myself and I",
  "BOXSIZEX": 9.2874,
  "BOXSIZEY": 9.2874,
  "BOXSIZEZ": 9.2874,
  "BOXTYPE": "Triclinic",
  "CHAINNAMES": Object {
    "A": "Spike",
    "B": "Spike",
    "C": "Spike",
    "G": "Glycans",
    "H": "Glycans",
    "I": "Glycans",
  },
  "CL": 0,
  "DOMAINS": Array [],
  "DPPC": 0,
  "ENSEMBLE": "NPT",
  "FF": "CHARMM36 All-atom additive",
  "FREQUENCY": "100",
  "INTERFACES": Array [],
  "LENGTH": "411.7",
  "LIGANDS": Array [],
  "MEMBRANE": "No",
  "NA": 0,
  "NAME": "This is a test",
  "PCOUPLING": "Isotropic",
  "PROT": 1138,
  "PROTATS": 9106,
  "SNAPSHOTS": 4117,
  "SOL": 0,
  "SYSTATS": 9727,
  "TEMP": 310,
  "TIMESTEP": "2",
  "WAT": "TIP3P",
}
`;
